api/v1beta1/octavia_types.go

```diff
@@ -366,7 +369,7 @@ func SetupDefaults() {
 		HealthManagerContainerImageURL: util.GetEnvVar("RELATED_IMAGE_OCTAVIA_HEALTHMANAGER_IMAGE_URL_DEFAULT", OctaviaHealthManagerContainerImage),
 		WorkerContainerImageURL:        util.GetEnvVar("RELATED_IMAGE_OCTAVIA_WORKER_IMAGE_URL_DEFAULT", OctaviaWorkerContainerImage),
 		ApacheContainerImageURL:        util.GetEnvVar("RELATED_IMAGE_OCTAVIA_APACHE_IMAGE_URL_DEFAULT", ApacheContainerImage),
-		OctaviaAPIRouteTimeout:         util.GetEnvVar("OCTAVIA_API_TIMEOUT", APITimeout),
+		OctaviaAPIRouteTimeout:         util.GetEnvVar("OCTAVIA_API_TIMEOUT", fmt.Sprintf("%d", APITimeout)),
```

Comment:
Why do we need the environment variable for this? You're adding it to the Spec right?
https://github.com/openstack-k8s-operators/octavia-operator/pull/403/files#diff-15ba2b697b963cd8e176ef9dd251f4cad85eeada22d61875a64f90b139b609b0R212

Wouldn't this be configured via the spec rather than build time environment variables?

Reply:
make sense, now setting by spec could be removed there.